std/file.d

```diff
@@ -886,11 +886,20 @@ version (Windows) private void writeImpl(scope const(char)[] name, scope const(F
 }
 
 /***************************************************
- * Rename file `from` _to `to`.
+ * Rename file `from` _to `to`, moving it between directories if required.
  * If the target file exists, it is overwritten.
+ *
+ * The operation is atomic, which means, that it is not possible to
+ * rename a file across different mount points or drives. If `to` already exists, it
+ * will be replaced atomically, which means, that during the operation, there
+ * is no time period, when `to` is missing. See
+ * $(HTTP man7.org/linux/man-pages/man2/rename.2.html, manpage for rename)
+ * for more details.
```

Comment:
English is my third language but it seems to me that there are too many commas here.

Specifically the ones in "which means, that" and "time period, when" sound redundant to me.

Reply:
Nit: s/Clearify/Clarify/ in commit message

Reply:
Commas are not my strength... :-)